src/pk_var.h

```diff
@@ -139,6 +139,9 @@
 #define IS_OBJ(value)   ((value & _MASK_OBJECT) == _MASK_OBJECT)
 #define IS_OBJ_TYPE(var, obj_type) IS_OBJ(var) && AS_OBJ(var)->type == obj_type
 
+// Check special type case
+#define IS_NUM_BYTE(num) ((CHAR_MIN <= (num)) && ((num) <= CHAR_MAX))
+
```

Comment:
This is a very specific case, and the macro doesn't have anything to do with `Var`. As I've mentioned in my last review ...

> Even if we're using this as a function (instead of a macro), that shouldn't be public, but a static function. Remove this.

This shouldn't be a public macro too. And for now, we don't need a macro (because we're not reusing this specific validation), just write it inside the `if`.

Reply:
Okay I see now. What you mean is we don't it to be made available for reusability. Not sure how this will be reused in the future but if it happens, we'll extract it and make it public. Thanks!